src/models/receipt.ts

```diff
@@ -0,0 +1,26 @@
+type aReceipt = {
+  id: string;
+  amount: number;
+  date: string;
+  isTest: boolean;
+  token: string;
+};
```

Comment:
Looking at your demo data, it looks like you're missing `donor`, `paymentMethod`, and `status`